bakery/src/tasks/validate-xhtml.js

```diff
@@ -28,8 +28,15 @@ const task = (taskArgs) => {
         args: [
           '-cxe',
           dedent`
-          collection_id="$(cat book/collection_id)"
-          for xhtmlfile in "${inputSource}/$collection_id/"${inputPath}
+          from_archive="$(cat book/server)"
+          if [[ -n $from_archive ]]
```

Comment:
consider passing in archive vs. git to the task itself, similar to [here](https://github.com/openstax/output-producer-service/blob/0fdd4bd61acb4231f263ee0a61bc44cd50616e02/bakery/src/tasks/look-up-book.js#L26), so pipeline behavior is always consistent from job to job.